cw/cw305/mix_columns_capture.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Mix column HD Attack - Capture portion
+
+Attack implemented by ChipWhisperer:
+Repo: https://github.com/newaetech/chipwhisperer-jupyter/blob/master/experiments/MixColumn%20Attack.ipynb
+Reference: https://eprint.iacr.org/2019/343.pdf
+
+See mix_columns_cpa_attack.py for attack portion.
+"""
+import chipwhisperer as cw
+import yaml
+
+import capture_traces as cp
+
+if __name__ == "__main__":
+  with open('capture.yaml') as f:
+    cfg_file = yaml.load(f, Loader=yaml.FullLoader)
+  ot = cp.initialize_capture(cfg_file['device'], cfg_file['spiflash'])
+  ot.target.output_len = cfg_file['capture']['plain_text_len_bytes']
+
+  # Key and plaintext generator
+  ktp = cw.ktp.VarVec()
```

Comment:
This should be `cw.ktp.Basic()` I think. At least there is no `VarVec()` in ChipWhisperer v.5.3.1.

Reply:
See comment below.